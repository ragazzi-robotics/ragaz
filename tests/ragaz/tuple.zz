def main():

    var a: int, b, c = 1, 2, 3
    print(a + b + c)

    a, b, c = 3, 2 + 2, 0
    print(a + b + c)

    var e = ("avocado", "banana", "orange")
    var f = ("avocado", "banana", "orange")
    print(e == f)
    f[0] = "pineaple"
    print(e == f)

    print(e)
