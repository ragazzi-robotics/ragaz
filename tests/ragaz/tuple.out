6
7
True
False
('avocado', 'banana', 'orange', )
